src/main.c

```diff
@@ -113,6 +115,15 @@ static void parse_options(int argc,
                 // }
                 break;
             }
+            case 'i':
+            {
+                arg_initrd_file = optarg;
+                // if (arg_fw_file)
```

Comment:
Please remove the commented code here. I know I am doing that on my own on a lot of places, but we should not introduce new ones.